package.json

```diff
@@ -96,6 +96,7 @@
     "heroku-postbuild": "run-s --silent build",
     "analyze": "cross-env ANALYZE=true LONG_CACHE=false BASE_URL=https://img.shields.io npm run build",
     "start:server": "cross-env HANDLE_INTERNAL_ERRORS=false RATE_LIMIT=false node server 8080 ::",
+    "start:server:debug": "cross-env ALLOWED_ORIGIN=http://localhost:3000 HANDLE_INTERNAL_ERRORS=false RATE_LIMIT=false node --nolazy --inspect=9229 server 8080 ::",
```

Comment:
was this an intentional add? I'm wondering if `9229` is a process id and how dynamic that value will be

Reply:
Yes. `9229` is a port. I was following the [VS Code Debugging instructions](https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_launch-configuration-support-for-npm-and-other-tools), I figure others would find it useful. I can revert this if you'd like.

Reply:
I was just wondering whether this would be user/machine specific. If it isn't and everyone can run it as-is on his or her machine then 👍, but if not then we should prob remove

Reply:
Unless you're debugging another app on that same port it should work OOTB, you just have to add the debug profile in VSCode's `launch.json`:

```json
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
      {
        "type": "node",
        "request": "launch",
        "name": "Launch via NPM",
        "cwd": "${workspaceFolder}",
        "runtimeExecutable": "npm",
        "runtimeArgs": [
          "run-script", "start:server:debug"
        ],
        "port": 9229
      }
    ]
}
```

Reply:
Could this be documented somewhere in the contributing guidelines? And if it's vscode-specific, could that be reflected in the script name? Perhaps `start:server:vscode-debug` or something like that.

Reply:
We're also in the process of adding a vs code `extensions.json` file to provided recommended extensions in #2581 so perhaps we could do all of the above :

* update the script name to include vscode as suggested
* include the `.vscode/launch.json` file in the repo (update the gitignore with a negated pattern for this or change the excluded path)
* update contributor docs with relevant info (this could be done in future work) on the script/launch settings and extensions

Reply:
@paulmelnikow Do you want the documentation / task in this PR or a different PR?